coverage_ai/AICaller.py

```
@@ -3,11 +3,31 @@
 import time
 
 import litellm
+from functools import wraps
 from wandb.sdk.data_types.trace_tree import Trace
+from tenacity import retry, retry_if_exception_type, retry_if_not_exception_type, stop_after_attempt, wait_fixed
```

Ruff

`tenacity.retry_if_exception_type` imported but unused

Remove unused import

(F401)